.github/workflows/main.yml

```diff
@@ -25,20 +25,19 @@ jobs:
         # Version Spec of the version to use.  Examples: 10.x, 10.15.1, >=10.15.0
         node-version: 12.x
 
-    - name: Setup Emscripten tools
-      uses: justjake/setup-emscripten@master
-      with:
-        emscripten-version: '1.39.18'
-        emsdk-version: '1.39.18'
-
-    - name: Install Yarn
-      run: 'npm install -g yarn@1.22.4'
-
     # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
     - uses: actions/checkout@v2
 
+    - name: Cache dependencies
+      uses: actions/cache@v2
+      with:
+        path: "**/.yarn/cache"
+        key: dependency-cache
```

Comment:
Should this include the checksums of various yarn.lock files? Or some other checked in yarn files? Won’t using a fixed cache key cause the cache to grow forever?

Reply:
The action seems to suggest using `${{ hashFiles('**/lockfiles') }}` in the key - will try that.